sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,10 +1,23 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
```

Comment:
@weshaggard would our build and release pipeline be able to handle this version jump from `1.0.0-beta.4` to `12.0.0-beta.5`?